From 3ad8739bd8a4f3b58fe591c89a5f209bf2eb60b1 Mon Sep 17 00:00:00 2001
From: "Christian Jaeger (dull)" <ch@christianjaeger.ch>
Date: Fri, 19 Nov 2021 17:18:14 +0000
Subject: .chj-home/.xscreensaver 000: [PATCH 134/191] .xscreensaver: blank and off fast

Because with the old settings it would, crazily, run screensavers
(concluded from CPU load) even after 35-45' of inactivity.

This was/is ever unclear, are those times *after* each other? Not all
from the start of inactivity?
---
 .chj-home/.xscreensaver | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/.chj-home/.xscreensaver b/.chj-home/.xscreensaver
--- a/.chj-home/.xscreensaver
+++ b/.chj-home/.xscreensaver
@@ -1,5 +1,5 @@
 # XScreenSaver Preferences File
-# Written by xscreensaver-demo 5.45 for chris on Thu Sep 23 17:10:59 2021.
+# Written by xscreensaver-demo 5.45 for chris on Fri Nov 19 17:17:27 2021.
 # https://www.jwz.org/xscreensaver/
 
 timeout:	0:10:00
